content/en/getting_started/test_impact_analysis/_index.md

```diff
@@ -0,0 +1,157 @@
+
+You can create [dashboards][6] to visualize your testing metrics, or use an [out-of-the-box dashboard][7] containing widgets populated with data collected by the Test Impact Analysis to help you identify areas of improvement with usage patterns and trends. 
+
+## Examine results in the Test Optimization Explorer
+
+The [Test Optimization Explorer][8] allows you to create visualizations and filter test spans using the data collected from Test Optimization and the Test Impact Analysis. When the Test Impact Analysis is active, it displays the amount of time saved for each test session or commit. The duration bars turn purple to indicate active test skipping.
+
+{{< tabs >}}
+{{% tab "Session" %}}
+
+Navigate to [**Software Delivery** > **Test Optimization** > **Test Runs**][101] and select `Session` to start filtering your test session span results.
+
+{{< img src="/getting_started/intelligent_test_runner/itr_sessions.png" alt="Test session results in the Test Optimization Explorer filtered on tests skipped by the Test Impact Analysis" style="width:100%" >}}
+
+[101]: https://app.datadoghq.com/ci/test-runs?query=test_level%3Asession
+
+{{% /tab %}}
+{{% tab "Module" %}}
+
+Navigate to [**Software Delivery** > **Test Optimization** > **Test Runs**][101] and select `Module` to start filtering your test module span results. 
+
+{{< img src="/getting_started/intelligent_test_runner/itr_modules.png" alt="Test module results in the Test Optimization Explorer filtered on tests skipped by the Test Impact Analysis" style="width:100%" >}}
+
+[101]: https://app.datadoghq.com/ci/test-runs?query=test_level%3Amodule
+
+{{% /tab %}}
+{{% tab "Suite" %}}
+
+Navigate to [**Software Delivery** > **Test Optimization** > **Test Runs**][101] and select `Suite` to start filtering your test suite span results. 
+
+{{< img src="/getting_started/intelligent_test_runner/itr_suites.png" alt="Test suite results in the Test Optimization Explorer filtered on tests skipped by the Test Impact Analysis" style="width:100%" >}}
+
+[101]: https://app.datadoghq.com/ci/test-runs?query=test_level%3Asuite
+
+{{% /tab %}}
+{{% tab "Test" %}}
+
+Navigate to [**Software Delivery** > **Test Optimization** > **Test Runs**][101] and select `Test` to start filtering your test span results. 
+
+{{< img src="/getting_started/intelligent_test_runner/itr_tests.png" alt="Test results in the Test Optimization Explorer filtered on tests skipped by the Test Impact Analysis" style="width:100%" >}}
+
+[101]: https://app.datadoghq.com/ci/test-runs?query=test_level%3Atest
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Use the following out-of-the-box Test Impact Analysis [facets][9] to customize the search query:
+
+Code Coverage Enabled
+: Indicates whether code coverage tracking was active during the test session.
+
+Skipped by ITR
```

Comment:
It's not going to be `ITR` anymore, right? Maybe we want to add some note in the documentation clarifying that this used to be the old name and it's still used in some places (e.g., here and in tags), but I'm not sure what is the best way to do it.

Reply:
@KickingTheTV can you advise?

Reply:
Yeah, I'm not sure how we want to approach this, but maybe a general note at the beginning of the docs for Test Impact Analysis? Or something more targeted around these various tag names and filter facets that are being generated.

UI-wise we're changing the name to Test Impact Analysis. But for our tags, they will keep `itr` in their naming for likely another 2 quarters. Starting next quarter we'll come out with new tags with the new name `tia`, and once we have 3 months of history with the new tag and old tag, we'll migrate over all of the notebooks and dashboards, so it should be a seamless experience